packages/auto_mappr/lib/src/builder/assignments/nested_object_mixin.dart

```diff
@@ -50,18 +50,21 @@ mixin NestedObjectMixin on AssignmentBuilderBase {
 
     // Unknown mapping.
     if (nestedMapping == null) {
-      final sourceName = assignment.sourceField?.getDisplayString(withNullability: true);
+      //final emittedTarget = EmitterHelper.current.typeReferEmitted(type: target);
+      final sourceParentClass = assignment.sourceField?.enclosingElement.name;
+      final targetParentClass = assignment.targetField?.enclosingElement.name;
+      final enclosingMappingMessage = "Parent mapping holding this is '$sourceParentClass' -> '$targetParentClass'";
 
       if (target.isNullable) {
-        log.warning("Can't find nested mapping '$assignment' but target is nullable. Setting null");
+        log.warning(
+          "Can't find nested mapping '$assignment' but target is nullable. Setting null. $enclosingMappingMessage",
```

Comment:
Maybe also wrap it in `(...)` brackets? To align it with how map_assignment_builder outputs thigs.